### PR TITLE
Update logging on rpc sending and receiving

### PR DESCRIPTION
**Pull request by m-samiec**
#### Description
Improved logging during rpcs. 
Also caught edge case where transient blueprint outers were having schema generated for them.

#### Release note
Feature: Improved logging during rpcs.

#### Primary reviewers
@joshuahuburn @BenNaccarato

### CODE REVIEW

File: SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp [hunk 1]
```diff
@@ -84,17 +84,11 @@ void USpatialReceiver::LeaveCriticalSection()
 		HandleActorAuthority(PendingAuthorityChange);
 	}
 
-	for (Worker_EntityId& PendingRemoveEntity : PendingRemoveEntities)
-	{
-		RemoveActor(PendingRemoveEntity);
```
Comment (BenNaccarato) [inline SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp]: Why are we no longer removing actors here? Has it been moved elsewhere

Comment (BenNaccarato) [reply SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp]: Discussed offline

Comment (m-samiec) [reply SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp]: For context, PendingRemoveEntities wasn't used, remove ops don't come down in critical sections.

File: SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp [hunk 2]
```diff
@@ -461,11 +461,17 @@ void USpatialSender::SendRPC(TSharedRef<FPendingRPCParams> Params)
 
 			if (Params->Function->HasAnyFunctionFlags(FUNC_NetReliable))
 			{
-				UE_LOG(LogSpatialSender, Verbose, TEXT("Send command request (entity: %lld, component: %d, command: %d)"), EntityId, CommandRequest.component_id, Schema_GetCommandRequestCommandIndex(CommandRequest.schema_type));
+				UE_LOG(LogSpatialSender, Verbose, TEXT("Send reliable command request (entity: %lld, component: %d, function: %s, attempt: %s)"),
```
Comment (joshuahuburn) [inline SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp]: Minor nit: shouldn't this be "Sending"

File: SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SpatialGDKEditorSchemaGenerator.cpp [hunk 3]
```diff
@@ -400,7 +400,8 @@ TArray<UClass*> GetAllSupportedClasses()
 		if (SupportedClass->GetName().StartsWith(TEXT("SKEL_"), ESearchCase::CaseSensitive)
 			|| SupportedClass->GetName().StartsWith(TEXT("REINST_"), ESearchCase::CaseSensitive)
 			|| SupportedClass->GetName().StartsWith(TEXT("TRASHCLASS_"), ESearchCase::CaseSensitive)
-			|| SupportedClass->GetName().StartsWith(TEXT("HOTRELOADED_"), ESearchCase::CaseSensitive))
+			|| SupportedClass->GetName().StartsWith(TEXT("HOTRELOADED_"), ESearchCase::CaseSensitive)
+			|| SupportedClass->GetName().StartsWith(TEXT("PROTO_BP_"), ESearchCase::CaseSensitive))
```
Comment (joshuahuburn) [inline SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SpatialGDKEditorSchemaGenerator.cpp]: What's this addition exactly? Generated BPs?

Comment (m-samiec) [reply SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SpatialGDKEditorSchemaGenerator.cpp]: Yes. @Vatyx encountered it when experimenting with sublevels. See `BlueprintNodeTemplateCacheImpl::MakeCompatibleBlueprint` for more context.

**Pull request merged**